src/AutoRest.CSharp/Common/Input/InputTypes/Examples/InputClientExample.cs

```diff
@@ -5,5 +5,10 @@
 
 namespace AutoRest.CSharp.Common.Input.Examples
 {
-    internal record InputClientExample(InputClient Client, IReadOnlyList<InputParameterExample> ClientParameters);
+    /// <summary>
+    /// Set of client parameter sample values of a specific type
+    /// </summary>
+    /// <param name="Key">Type of a sample. E.g.: short version sample, all parameters sample, etc.</param>
```

Comment:
I believe the `key` here should be `key of the example`?
Just like what we have in swaggers:
```
"/request" : {
    "get" : {
	"x-ms-examples": {
	    "the key of example": "file path of the example"
	}
    }
}
```
maybe we could refer that concept here so that it would be more comprehensive

Reply:
Done